docker/llvm.dockerfile

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM pennylane/base:latest AS data-image
+
+FROM ubuntu:latest AS compile-image
+
+# Setup and install Basic packages
+RUN apt-get update && apt-get install -y apt-utils --no-install-recommends
```

Comment:
```suggestion
RUN apt-get update && apt-get upgrade -y && apt-get install -y apt-utils --no-install-recommends
```

Reply:
Maybe this should also change on the pennylane.dockerfile?  https://github.com/pennylaneAI/pennylane/blob/master/docker/pennylane.dockerfile#L18=

Reply:
Yep, great idea. Updating both here makes a lot of sense.